oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/config/group/openapi/EndpointGroupingRule4Openapi.java

```diff
@@ -34,7 +34,7 @@ void addDirectLookup(String serviceName, String endpointName, String endpointGro
         endpointNameLookup.put(endpointName, endpointGroupName);
     }
 
-    void addGroupedRule(String serviceName, String endpointGroupName, String ruleRegex) {
+    public void addGroupedRule(String serviceName, String endpointGroupName, String ruleRegex) {
```

Comment:
Exposing new method is not recommended usually. I think this method is only used in benchmark initialization stage.